prune.go

```diff
@@ -159,23 +163,33 @@ func (p Pruner) Prune() (*Stats, error) {
 			stats.SizeRemoved += s.SizeRemoved
 		}
 
-		// remove and skip dir
+		// spawn fo routine to do the removal
+		wg.Add(1)
+		go p.remove(path, info, &wg, errorChan)
```

Comment:
could even move this part above and do the stats(atomic inc the stats value then though) counting in goroutine itself ? 
but before that trying to see if any of this makes sense/adds value ?

Reply:
hmm I think a goroutine per file is a big excessive maybe. I wonder if just launching one per cpu, maybe the performance would improve

Reply:
Maybe something like a threadpool?

Reply:
done, capped by cpu